nbs/00_utils.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from fastcore.test import *
</pre><p>This should be on a separate cell (so we don't export everything in fastcore.test from this module)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/tsfeatures/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>